python/Ryu.py

```diff
@@ -564,8 +1094,29 @@ def process_flow_stats(self, stats, dp):
             wildcards = stat.match.wildcards
             del match['dl_dst']
             del match['dl_src']
-            del match['dl_type']
-            del match['wildcards']
+            del match['wildcards'] 
+            if match['dl_type'] == 34525:
+                if src_mask > 0:
```

Comment:
we can't match on IPv6 in 1.0 so I guess I don't fully understand what this is attempting to do?  Can you explain?